Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -143,12 +154,33 @@ public static void PurgeMediaCacheDirectly(this UITestContext context, bool togg
         /// <c>Lombiq.Tests.UI.Shortcuts</c> enabled.
         /// </summary>
         public static void ExecuteRecipeDirectly(this UITestContext context, string recipeName) =>
-            context.GoToRelativeUrl("/Lombiq.Tests.UI.Shortcuts/Recipe/Execute?recipeName=" + recipeName);
+            GoTo<RecipeController>(
+                context,
+                nameof(RecipeController.Execute),
+                (nameof(recipeName), recipeName));
+
+        /// <summary>
+        /// Navigates to a page whose action method throws <see cref="InvalidOperationException"/>. This causes ASP.Net
+        /// Core to display an error page.
+        /// </summary>
+        public static void GoToErrorPageDirectly(this UITestContext context) => GoTo<ErrorController>(context);
+
+        private static void GoTo<TController>(
+            UITestContext context,
```

Comment:
Wouldn't it be desirable to add `this` here and invoke `context.GoTo<TController>()` as an extension method?

Reply:
I agree, indeed.

Reply:
Hmm, I never use extension methods in the same class they are defined. It make it clearer that this is a local method. If you think that's not logical I'm not overly attached to that practice. Just want to share my rationale for not doing it that way.

Reply:
I don't have a strong feeling either way but an extension does "look" nicer to me.

Reply:
@DAud-IcI Why is it important for you to make it clear that something is a local method? I'm asking out of curiosity.

Reply:
After sleeping on it, I think I explained myself poorly. Its better to say that when I use extension method I expect it to not be local because it usually isn't. So when it is, that's confusing. Additionally when I see a private extension method, that's an instant WTF moment for me and have to do a double take to see if that's a mistake or what.

Reply:
That last point convinces me.

I'm just wondering - why would you NOT make it public in the first place?

Reply:
Because it's highly specific to the shortcuts module that we only invoke through these other extensions in this class. I plan to make a more general version and that can be public.

Reply:
OK. Thank you for elaborating on this. I appreciate it.

Reply:
Let's leave it like then.